src/TBuffer.cpp

```diff
@@ -3051,8 +3052,7 @@ void TBuffer::append(const QString& text, int sub_start, int sub_end, TChar form
         // The ternary operator is used here to set/reset only the TChar::Echo bit in the flags:
         TChar c(format.mFgColor,
                 format.mBgColor,
-                (mEchoText
-                 ? (TChar::Echo | (format.mFlags & TChar::TestMask))
+                (mEchoingText ? (TChar::Echo | (format.mFlags & TChar::TestMask))
```

Comment:
Was the first and second parts of the ternary ( `<condition> ? <true-value> : <false-value>` ) put back on one line by clang-format? I'd originally laid it out on three lines to clearly split the longer parts in a logical way but it might not have liked that I suppose in here and the next change place.

*\*sigh\* Ah well, dems da (line) breaks....*